cmd/init-repo/main.go

```diff
@@ -68,10 +70,26 @@ func main() {
 	if *repoOrg != "" {
 		cfg.SourceOrg = *repoOrg
 	}
+	if *githubHost != "" {
+		cfg.GithubHost = *githubHost
+	}
+	if *basePackage != "" {
+		cfg.BasePackage = *basePackage
+	}
 
-	if cfg.SourceRepo != "kubernetes" {
-		BaseRepoPath = filepath.Join(SystemGoPath, "src", "github.com", cfg.TargetOrg)
+	if cfg.GithubHost == "" {
```

Comment:
put this into an else above

Reply:
@sttts: if we put this in the `else` branch above, there is no way to specify the value in config file.

Reply:
sorry, misread.